.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+updates:
+
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "08:00"
+      timezone: "America/Los_Angeles"
+    target-branch: "main"
+    commit-message:
+      prefix: "[golang-version]"
+      include: "scope"
+    pull-request-branch-name:
+      separator: "-"
+    open-pull-requests-limit: 8
+    reviewers:
+      - "Layr-Labs/eigenda-intg"
```

Comment:
can you add a comment with link to https://github.com/orgs/Layr-Labs/teams/eigenda-intg
Also can we remove hope from that team and add ethen and bowen?

Reply:
done